VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/hearing-controls/hearing-controls-base.component.ts

```diff
@@ -36,26 +37,23 @@ export abstract class HearingControlsBaseComponent implements OnInit, OnDestroy
     selfViewOpen: boolean;
     displayConfirmPopup: boolean;
 
-    protected constructor(protected videoCallService: VideoCallService, protected eventService: EventsService, protected logger: Logger) {
+    protected constructor(
+        protected videoCallService: VideoCallService,
+        protected eventService: EventsService,
+        protected deviceTypeService: DeviceTypeService,
+        protected logger: Logger
+    ) {
         this.handRaised = false;
         this.remoteMuted = false;
         this.selfViewOpen = false;
         this.displayConfirmPopup = false;
     }
 
     get canShowScreenShareButton(): boolean {
-        const allowedRoles = [Role.Representative, Role.JudicialOfficeHolder, Role.Judge];
-        if (allowedRoles.includes(this.participant.role)) {
-            return true;
-        }
+        const isNotTablet = !this.deviceTypeService.isTablet();
+        const isAllowedRole = this.participant?.hearing_role !== HearingRole.WITNESS;
```

Comment:
the ticket requests VIH-6905
As a: Hearing Participant (Judge, JoH, Litigant in Person, Representative, Witness)
This looks like everyone but a witness can screen share.
If you want to stop all observers, that's a hearing role in it self

Reply:
The ticket changed AC - now, anyone who isn't a witness is allowed to screen share